com.jvieira.tpt.Metronome.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.jvieira.tpt.Metronome.desktop</id>
```

Comment:
```suggestion
  <id>com.jvieira.tpt.Metronome</id>
  <launchable type="desktop-id">com.jvieira.tpt.Metronome.desktop</launchable>
```

Reply:
The appdata should be upstreamed if possible

Reply:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Thank you! I will upstream this changes.